docs/baremetal-agent-controller.md

```diff
@@ -47,17 +49,40 @@ The config above should provide you with an environment that is ready to be used
 assisted installer, and BMAC tests. Here are a few tips that would help simplifying the environment
 and the steps required:
 
-- Clone [baremetal-operator][bmo] somewhere and set the BAREMETAL_OPERATOR_LOCAL_IMAGE in your config.
+- Clone [baremetal-operator][bmo] somewhere and set the `BAREMETAL_OPERATOR_LOCAL_IMAGE` in your config.
 
 Once dev-script is up and running, modify the worker(s) and add 2 more disks (10GB should be enough)
-as they are required by the Assisted Installer Operator.
+as they are required by the Local Storage Operator.
+
+You can use the following snippet to attach a disk to the libvirt VM
+
+```bash
+export CLUSTERNAME=ostest
+export VMNAME=master_0
+export DISK=sdb
+
+qemu-img create -f raw /home/dev-scripts/pool/${VMNAME}_manual_${DISK}.img 10G
+
+virsh attach-disk ${CLUSTERNAME}_${VMNAME} \
+--source /home/dev-scripts/pool/${VMNAME}_manual_${DISK}.img \
+--target ${DISK} \
+--persistent
+```
```

Comment:
I have a small counter-proposal for this. What do you think about taking @lranjbar dev-script PR over so that we won't need to do this? https://github.com/openshift-metal3/dev-scripts/pull/1230